dynawo/sources/API/CRT/CRTXmlHandler.h

```diff
@@ -55,7 +55,48 @@ class ElementWithIdHandler : public xml::sax::parser::ComposableElementHandler {
   void create(attributes_type const& attributes);
 
  private:
-  std::string cmpRead_;  ///< current component
+  std::string idRead_;  ///< current element id
+};
+
+/**
+ * @class ComponentHandler
+ * @brief Handler used to parse a component element
+ */
+class ComponentHandler : public xml::sax::parser::ComposableElementHandler {
+ public:
+  /**
+   * @brief Constructor
+   * @param root_element complete name of the element read by the handler
+   */
+  explicit ComponentHandler(elementName_type const& root_element);
+
+  /**
+   * @brief default destructor
+   */
+  ~ComponentHandler() { }
```

Comment:
I know it's the same everywhere in the file, but the destructor should be virtual

Reply:
Yes, it was a copy/paste of `ElementWithIdHandler`
Fixed